include/libvfio-user.h

```diff
@@ -57,6 +57,8 @@ extern "C" {
 #define LIB_VFIO_USER_MAJOR 0
 #define LIB_VFIO_USER_MINOR 1
 
+#define VFU_DMA_REGIONS  0x10
```

Comment:
I know it is not related to this patch, but from where this limit came. Maybe @tmakatos knows.

Reply:
It's what QEMU typically uses in our tests, we should definitely find a better way of determining the number of DMA regions.

Reply:
#202